### ISSUE TITLE
VLCTube breaks chatting in comments in Firefox 33.0.2

### ISSUE THREAD

**Issue by michaelmotes** [jackun/VLCTube#58]
I use Firefox 33.0.2 and when I use VLCTube during a live stream the chat does not work. I can see chat happening, but when I press enter to comment in chat, I get this in the log:
````
TypeError: c.getCurrentVideoConfig is not a function
https://s.ytimg.com/yts/jsbin/www-en_US-vflTLkWXB/common.js
Line 769
````

**Comment by michaelmotes** [comment]
Resolved in commit ce15ec6

Thanks

**Comment by jackun** [comment]
https://github.com/jackun/VLCTube/commit/e994c314284c2c509dca5cd64318434b6db5b893
Does this break chatting? Commenting seemed to work, but...
